style.css

```diff
@@ -0,0 +1,147 @@
+:root {
+  --color-main_theme: #36b37e;
+  --color-second_green: #c5e9da;
+  --color-light-green: #ebf0ee;
+  --color-dark-blue: #172b4d;
+  --color-white-bg: #e5e5e5;
```

Comment:
- [ ] Please make sure to follow the background color present in the design guidelines for styling ✨ The background color used in the template is `#FFFFFF` ⚒ 

Your design | Design guidelines
---| ---|
![image](https://user-images.githubusercontent.com/106119105/234726647-fb573efd-0f32-4d4e-b308-aff9c4ac55fb.png) | ![image](https://user-images.githubusercontent.com/106119105/234726672-06f4a649-7da1-4233-b4d8-4e9c658274ad.png)

Reply:
Yes, so valid. Thank you very much!